.prow/config.yaml

```diff
@@ -50,132 +50,56 @@ tide:
 # presubmits list Prow jobs that run on pull requests
 presubmits:
   gojek/feast:
-  - name: unit-test-core
+  - name: test-core-and-ingestion
     decorate: true
     always_run: true
     spec:
-      volumes:
-      - name: service-account
-        secret:
-          secretName: prow-service-account
       containers:
       - image: maven:3.6-jdk-8
-        volumeMounts:
-        - name: service-account
-          mountPath: /etc/service-account
-          readOnly: true
-        env:
-        - name: GOOGLE_APPLICATION_CREDENTIALS
-          value: /etc/service-account/service-account.json
-        command: [".prow/scripts/run_unit_test.sh", "--component", "core"]
+        command: [".prow/scripts/test-core-ingestion.sh"]
```

Comment:
@davidheryanto Would it make more sense to move these scripts to `/tests` or `/infra` instead of prow?